docs/user/whats-new.asciidoc

```diff
@@ -90,55 +68,47 @@ image::images/metric-customization.png[Customization options for a metric panel]
 === Managing {kib} and data
```

Comment:
```suggestion
=== Alerting, cases, and connectors
```
IMO it should match the title we used in https://www.elastic.co/guide/en/kibana/8.15/whats-new.html#_alerting_cases_and_connectors